src/node/supervisor.rs

```diff
@@ -80,16 +79,16 @@ impl Supervisor {
                 Event::GetShardsNum => {
                     // TODO connect to scylla-shard-zero and get_cql_opt to finally get the shards_num
                     // for testing purposes, we will manually define it.
-                    self.shards_num = 1; // shard(shard-0)
+                    self.shard = 1; // shard(shard-0)
```

Comment:
I prefer to use shard_count here. For the naming of field, I prefer to use [struct name]_count for the counting, and use id2[struct name] for the hash map of ID to struct. How do you think?